### ISSUE TITLE
Questions and features

### ISSUE THREAD

**Issue by jottinger** [ChasTechProjects/Debian64Pi#5]
1) it would be nice to see the actual system dependencies required; I had to figure out what qemu dependencies were required. (This is OS-dependent; I'm running the installation on a pi4 itself.)
2) Does it resize the partition table on boot on a successful installation?
3) It got an error at the end of stage1.sh, saying `mkdir: cannot create directory ‘/tmp/pi-kernel’: File exists`. I haven't had a chance to try the build yet; this may be an outlier.

**Comment by TwinUsers** [comment]
1, I’ve never tried on a pi4 unfortunately so no help

2, it doesn’t auto resize partition, however it could be possible. If a 64bit raspi-config is built and .deb package is then installed during build you could edit cmdline.txt to request it. 

3, I’ve never had this issue, so could be raspberry pi related, I’ve always used a 64bit Ubuntu vps to build.

**Comment by AlessandroMelino** [comment]
I am trying to change to Debian 64 bits with a Raspberry Pi4 with buster, and when I run the stage 1 script I get the following error:

```
mkdir: cannot create directory ‘/mnt/lib/modules’: No such file or directory
Run stage2.sh in chroot for stage 2.
chroot: failed to run command ‘/bin/bash’: No such file or directory
```
I followed every step and I don't know what to do. If you can help me I would be very grateful.

Best regards.
Alessandro

**Comment by TwinUsers** [comment]
Are you running with root access and do you have wget & curl installed? @AlessandroMelino

**Comment by AlessandroMelino** [comment]
> Are you running with root access and do you have wget & curl installed? @AlessandroMelino

Yes, I am running with root access, as says the tutorial, but I don't know if I have curl installed, so I will see if I have it and try again.
